ext/drsyms/CMakeLists.txt

```diff
@@ -182,6 +182,10 @@ configure_extension(drsyms_static ON)
 configure_drsyms_target(drsyms_static)
 use_DynamoRIO_extension(drsyms_static drcontainers)
 
+# we always use the elftoolchain library when building with cmake
+append_property_string(TARGET drsyms COMPILE_FLAGS "-DDRSYM_HAVE_LIBELFTC")
+append_property_string(TARGET drsyms_static COMPILE_FLAGS "-DDRSYM_HAVE_LIBELFTC")
```

Comment:
Put this into configure_drsyms_target to avoid duplicating

Reply:
Done.